tests/test_integration.py

```diff
@@ -246,11 +246,13 @@ def test_regular_after_first_deps_only_is_supported():
     assert f"/.tox/{NATIVE_TOXENV}/bin/python" in lines[0]
 
     # check that "test" was not installed to current environment
+    shutil.rmtree("./test.egg-info")
```

Comment:
And here you don't care about the package being installed into the current dir? Or is just the egg-info being left over for some reason?

Reply:
the egg info in PWD fools pip freeze. it is not installed in current environment, it's just happens to be in the current dir.

Reply:
Ah, ok.